### DISCUSSION TITLE
Graduation Shipping Form

### DISCUSSION THREAD

**Discussion by holiq** [community/community#17758] [GitHub Education]
I have sent data on this [form](https://airtable.com/shrVMo8ItH4wjsO9f)  but haven't received an email, is it really like that?

**Comment by proarafat** [comment]
Yup, first you will get an email and then they Send You a PostCard to your Home.

**Reply by holiq** [reply]
so I just wait for the email or send the data again on the form?

**Reply by proarafat** [reply]
you don't need to do that thing.

**Reply by holiq** [reply]
Okay thank you
